wled00/xml.cpp

```diff
@@ -186,11 +186,21 @@ void sappends(char stype, const char* key, char* val)
   }
 }
 
+void addPinOwners() {
```

Comment:
Separate function because this information was added to multiple subpages.  Better to avoid code replication.

I'm happy to add the code that tracks that other pins are not usable.  I just don't realize they're missing.  What reserved pins are not yet defined / allocated?

Reply:
P.S. -- I added code to reserve pins 6-11 early in `setup()`.

Reply:
After I wrote those comments I went ahead and looked again at Pin Manager as a whole.
It looks like there is actually no need to allocate reserved pins since isPinOk() handles that via hard-coded values. And all other functions call isPinOk() so there is no possibility for user to be able to select one of those not get a false when calling isPinAllocated().